### ISSUE TITLE
doc/core/howto/listings/pb/pb1server.py needs reactor.run()

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11089]
|<img alt="mcmillen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| mcmillen reported|
|-|-|
|Trac ID|trac#1559|
|Type|defect|
|Created|2006-03-10 00:37:45Z|

The sample file:

doc/core/howto/listings/pb/pb1server.py

needs the line "reactor.run()" at the end. Otherwise the server immediately exits. The online PB tutorial (http://twistedmatrix.com/projects/core/documentation/howto/pb-usage.html) is also missing the line. (Presumably the listings there are autogenerated based on the contents of SVN trunk, however.)

<details><summary>Searchable metadata</summary>

```
trac-id__1559 1559
type__defect defect
reporter__mcmillen mcmillen
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1141951065000000 1141951065000000
changetime__1142096841000000 1142096841000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="foom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|foom set status to `closed`|
|-|-|

(In [16243]) Add reactor.run() to example. Fixes [#11089](https://github.com/twisted/twisted/issues/11089) reported by mcmillen.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
